CHANGELOG.unreleased.md

```diff
@@ -28,6 +28,7 @@
 - [Backup/Restore] Fix unnecessary pool selector in XO config backup restore modal [Forum#8130](https://xcp-ng.org/forum/topic/8130/xo-configbackup-restore) (PR [#7287](https://github.com/vatesfr/xen-orchestra/pull/7287))
 - [File restore] Fix potential race condition in partition mount/unmount (PR [#7312](https://github.com/vatesfr/xen-orchestra/pull/7312))
 - [Modal] Fix opened modal not closing when navigating to another route/URL (PR [#7301](https://github.com/vatesfr/xen-orchestra/pull/7301))
+- [PIF] Fix PIF reconfiguration is only triggered if IPv6 has been updated (PR [#7324](https://github.com/vatesfr/xen-orchestra/pull/7324))
```

Comment:
It's unclear if what you describe is the unwanted behavior that has been fixed or the new correct behavior.

Using past tense might make it clearer.